pkg/syncer/metadatasyncer.go

```diff
@@ -373,24 +381,12 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 				volumeInfoCrDeletionMap[vcconfig.Host] = make(map[string]bool)
 				volumeOperationsLock[vcconfig.Host] = &sync.Mutex{}
 			}
-			if clusterFlavor == cnstypes.CnsClusterFlavorVanilla {
```

Comment:
Removing the redundant clusterFlavor check as the code block is only for Vanilla. See Line:333